example/lib/phone_sign_in.dart

```diff
@@ -14,7 +14,10 @@ class PhoneAuth extends StatelessWidget {
         padding: const EdgeInsets.all(24.0),
         child: Column(
           children: [
-            const SupaPhoneAuth(redirectUrl: '/verify_phone'),
+            const SupaPhoneAuth(
+              phoneAuthAction: PhoneAuthAction.signIn,
+              redirectUrl: '/home',
+            ),
             TextButton(
               child: const Text(
                 'Already have an account? Sign In',
```

Comment:
The other screen elements must be changed as well.

I'd change it to
**"Don't have an account? Sign up"**